docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -1,26 +1,25 @@
 [id="adopting-the-object-storage-service_{context}"]
 
-= Adopting the Object Storage service
+= Adopting the {object_storage}
 
-This section only applies if you are using OpenStack Swift as {object_storage_first_ref}. If you are using the Object Storage API of Ceph RGW this section can be skipped.
+If you are using OpenStack Swift as the {object_storage_first_ref}, adopt the {object_storage} to the {rhos_long} environment. If you are using the Object Storage API of the Ceph Object Gateway (RGW), skip this procedure.
```

Comment:
@cschwede What is meant by, “If you are using OpenStack Swift as an Object Storage service”? 
Can we say, “If you are using Object Storage as a service" instead?